stbt_auto_selftest.py

```diff
@@ -77,7 +80,6 @@ def main(argv):
     subparsers.add_parser('validate', help='Run (and check) the auto-selftests')
 
     cmdline_args = parser.parse_args(argv[1:])
-    screenshots_root = cmdline_args.screenshots_root or "selftest/screenshots"
```

Comment:
I don't think so.  I use git as a reference point.  If you are in a subdirectory of your git repo and do `git add .` you expect the subdirectory to be added, not the whole repo.